common/wonder_upgrades/wc_wonders_historical_upgrades.txt

```diff
@@ -1,5 +1,148 @@
 #Last Modified: 7/11/2019
 
+upgrade_old_god_cthun_1 = {
+	picture = GFX_upgrade_icon_old_god_cthun
+
+	potential = { has_wonder_upgrade =  upgrade_old_god_cthun_1 }
+	allow = { }
+	active = { }
```

Comment:
Do the Old Gods give bonuses to non-Old Gods' followers?

Reply:
you lose the province if you gain the wonder with the upgrade if your top liege is not old gods worship

Reply:
> you lose the province if you gain the wonder with the upgrade if your top liege is not old gods worship

https://github.com/Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth/pull/845#issuecomment-667676247